catalog/ui/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.less

```diff
@@ -51,22 +45,4 @@
   .interaction-revert {
     display: block;
   }
-}
-
-@{customElementNamespace}search-interactions.is-text {
-  .interaction-type-text {
-    display: none;
-  }
-}
-
-@{customElementNamespace}search-interactions.is-basic {
```

Comment:
Why did we get rid of the quick "basic" and "advanced" selectors in the search-interactions menu?

Reply:
This was based on design discussions; these were moved under templates